README.md

```diff
@@ -36,11 +36,9 @@ If you need help with CAPZ, please visit the [#cluster-api-azure][slack] channel
 
 This provider's versions are compatible with the following versions of Cluster API:
 
-| | Cluster API `v1alpha3` (`v0.3.x`) | Cluster API `v1alpha4` (`v0.4.x`) | Cluster API `v1beta1` (`v1.0.x`) |
-|---|---|---|---|
-|Azure Provider `v0.4.x` | ✓ |  |  |
-|Azure Provider `v0.5.x` |  | ✓ |  |
-|Azure Provider `v1.0.x` |  |  | ✓ |
+| | Cluster API `v1beta1` (`v1.0.x`) |
```

Comment:
I wonder if it would make sense to reword this now that there is only one version